src/test/java/models/testGame.java~origin_master

```diff
@@ -1,60 +0,0 @@
-package models;
-
-import org.junit.Test;
-
-import static org.junit.Assert.*;
-
-/**
- * Created by michaelhilton on 1/25/16.
- */
-public class testGame {
-
-    @Test
-    public void testGameCreation(){
-        Game g = new Game();
-        assertNotNull(g);
-    }
-
-    @Test
-    public void testGameBuildDeck(){
-        Game g = new Game();
-        g.buildDeck();
-        assertEquals(52,g.deck.size());
-    }
-    
-
-    @Test
-    public void testGameStart(){
-        Game g = new Game();
-        g.buildDeck();
-        g.shuffle();
-        g.dealFour();
-        assertEquals(1,g.cols.get(0).size());
-        assertEquals(1,g.cols.get(1).size());
-        assertEquals(1,g.cols.get(2).size());
-        assertEquals(1,g.cols.get(3).size());
-    }
-
-    @Test
```

Comment:
I removed that whole file because it's one of the extra git generated files Yizheng pushed on accident